### PR TITLE
Container Builder: Document and add a prepare command for it

### PR DESCRIPTION
**Pull request by iddan**
Document the container builder package and add it to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,17 @@ Amplication Data Service Generator is the component that generates the server-si
 
 To use this package as a library or as a CLI with other projects, follow the instructions on the package page.
 
+### [Amplication Design System](./packages/amplication-data-service-generator/README.md)
+
+Amplication Data Service Generator is the component that generates the server-side code (models, Prisma client, REST API, authentication, and authorization filters) of the application built with Amplication. This package is used internally by Amplication server.
+
+To use this package as a library or as a CLI with other projects, follow the instructions on the package page.
```
Comment (yuval-hazaz) [inline README.md:67]: This text talks about DSG instead of Design System

File: README.md [hunk 2]
```diff
@@ -60,6 +60,17 @@ Amplication Data Service Generator is the component that generates the server-si
...
+
+### [Amplication Container Builder](./packages/amplication-container-builder/README.md)
```
Comment (yuval-hazaz) [inline README.md:69]: I think this is not "cooked" enough to be used as a separate package and maybe should be removed from the main README for now

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (iddan) [reply README.md:69]: It works perfectly, I don't see why not

Comment (iddan) [reply README.md:69]: But I don't have a strong opinion about it.

Comment (yuval-hazaz) [reply README.md:69]: I suggest keeping the readme simple for the current time

**Review by yuval-hazaz — APPROVED** [review]

**Pull request merged**